frontend/tests/snapshots/landing.test.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from "react"
+import { LandingPage} from "../../compositions/"
+import renderer from "react-test-renderer"
+
+it("renders Saved Searches page correctly", () => {
+  const tree = renderer.create(<LandingPage />).toJSON()
```

Comment:
Nit: use the render method from test-utils, it generates a more accurate snapshot. Take a look at profile.test.tsx

Reply:
ok fixed